Lib/_pyio.py

```diff
@@ -315,7 +315,7 @@ class IOBase(metaclass=abc.ABCMeta):
 
     This class provides dummy implementations for many methods that
     derived classes can override selectively; the default implementations
-    represent a file that cannot be read, written or seeked.
+    represent a file that cannot be read, written or sought.
```

Comment:
While I understand this change, I think we can use `seeked` here. One existing wording is https://developer.mozilla.org/en-US/docs/Web/API/HTMLMediaElement/seeked_event which relates to `seek` operations.